.github/workflows/test-mobile-e2e-reusable.yml

```diff
@@ -82,7 +76,7 @@ jobs:
             ~/.cocoapods
           key: ${{ runner.os }}-pods-${{ hashFiles('apps/ledger-live-mobile/ios/Podfile.lock') }}
       - name: Configure AWS credentials
-        uses: aws-actions/configure-aws-credentials@v4
+        uses: aws-actions/configure-aws-credentials@v3
```

Comment:
[ASK] Why downgrade here ?

Reply:
It seems to be a copy paste issue, I revert to v4, thanks!